src/discriminators/composeChildTC.ts

```diff
@@ -1,11 +1,27 @@
-import { ObjectTypeComposer } from 'graphql-compose';
+import { ObjectTypeComposer, ObjectTypeComposerRelationOpts } from 'graphql-compose';
 import type {
   DiscriminatorTypeComposer,
   ComposeWithMongooseDiscriminatorsOpts,
 } from './DiscriminatorTypeComposer';
 import { prepareChildResolvers } from './prepareChildResolvers';
 import { reorderFields } from './utils/reorderFields';
 
+function copyBaseTcRelationsToChildTc(
+  baseDTC: ObjectTypeComposer<any, any>,
+  childTC: ObjectTypeComposer<any, any>
+) {
+  const relations = baseDTC.getRelations();
+  const childRelations = childTC.getRelations();
+  Object.keys(relations).forEach((name) => {
+    if (childRelations[name]) {
+      return;
+    }
+    childTC.addRelation(name, relations[name] as any);
```

Comment:
Somehow `getRelations` returns a `ObjectTypeComposerRelationThunkMap ` which I think is wrong, because in graphql-compose I never see that thunks are created for relations (only for fields) unless someone directly writes `ObjectTypeComposer. _gqcRelations` outside of `graphql-compose`. I will report that to `graphql-composer`.